client/src/helpers/APICalls/getProfiles.ts

```diff
@@ -0,0 +1,29 @@
+import { FetchOptions } from '../../interface/FetchOptions';
+import { SitterProfileForList } from '../../interface/SitterProfileForList';
+
+const fetchOptions: FetchOptions = {
+  method: 'GET',
+  headers: { 'Content-Type': 'application/json' },
+  credentials: 'include',
+};
+
+const getFilteredProfile = async (from: Date, to: Date): Promise<SitterProfileForList[]> => {
+  const fromDayOfWeek = from.toLocaleDateString('en-us', { weekday: 'long' }).toLowerCase();
+  const toDayOfWeek = to.toLocaleDateString('en-us', { weekday: 'long' }).toLowerCase();
+
+  const fromMinutes = from.getHours() * 60 + from.getMinutes();
+  const toMinutes = to.getHours() * 60 + to.getMinutes();
+
+  // looks messy, but it needs to be done to pass a multi-dimensional object/array
+  const query = `${fromDayOfWeek}[from]=${fromMinutes.toFixed(0)}&${toDayOfWeek}[to]=${toMinutes.toFixed(0)}`;
+  const encodedQuery = encodeURI(query);
```

Comment:
Any chance we could possibly use something like https://developer.mozilla.org/en-US/docs/Web/API/URL ?